Index: src/main/java/com/spa/demo/frontend/Cassa/CheckID.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>package com.spa.demo.frontend.Cassa;\r\n\r\nimport com.spa.demo.frontend.Cassa.Utils.WindowHandlerUtils;\r\nimport javafx.event.ActionEvent;\r\nimport javafx.fxml.FXML;\r\nimport javafx.scene.Node;\r\nimport javafx.scene.control.TextField;\r\nimport javafx.stage.Stage;\r\n\r\nimport java.io.IOException;\r\n\r\npublic class CheckID {\r\n\r\n    private Node node;\r\n    @FXML\r\n    private TextField watchId;\r\n\r\n    @FXML\r\n    void getWatchInformations(ActionEvent event) {\r\n\r\n    }\r\n\r\n    @FXML\r\n    void backToMainPage(ActionEvent event) throws IOException {\r\n        node = (Node) event.getSource();\r\n        WindowHandlerUtils.BackToCassaMainPage(node);\r\n    }\r\n\r\n    public void initialize()\r\n    {\r\n\r\n    }\r\n\r\n\r\n}\r\n\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/spa/demo/frontend/Cassa/CheckID.java b/src/main/java/com/spa/demo/frontend/Cassa/CheckID.java
--- a/src/main/java/com/spa/demo/frontend/Cassa/CheckID.java	(revision 607a20308bb53b2e8465f4f4311e109d56ff4b8a)
+++ b/src/main/java/com/spa/demo/frontend/Cassa/CheckID.java	(date 1732388899446)
@@ -5,7 +5,6 @@
 import javafx.fxml.FXML;
 import javafx.scene.Node;
 import javafx.scene.control.TextField;
-import javafx.stage.Stage;
 
 import java.io.IOException;
 
